tagmanager/src/tm_source_file.h

```diff
@@ -14,14 +14,12 @@
 #include <glib.h>
 #include <glib-object.h>
 
+#include "tm_parser.h"
```

Comment:
hum, this actually breaks every plugins because *tm_parser.h* isn't installed, so it gives nice errors like  `fatal error: tm_parser.h: No such file or directory`